.github/actions/release-connect/action.yml

```diff
@@ -102,6 +102,7 @@ runs:
         aws s3 sync --delete tmp_build_directory/ "${DEPLOY_PATH}"
 
     - name: Build connect-examples webextension
+      if: inputs.buildArtifacts == 'true'
```

Comment:
Don't you have to define the `inputs` in the section `inputs` at the top of the action? https://github.com/trezor/trezor-suite/pull/12448/files#diff-997d2c0811918f861e8d73a45635dc4a734155cc91e3b16b8fcd6e7b632e8d70R3

Also I think if you define it as `boolean` you don't need to treat it as a string. As we are doing in https://github.com/trezor/trezor-suite/blob/develop/.github/workflows/template-connect-test-params.yml#L22

Or is it for some reason I am overlooking better to do it like that?

Reply:
yes youre right, i like to use it as string instead fo bool because you are fixed in how the value is compared to bool where it can be `0` or `1` and `true` or `false` in different envs.